celery/app/task.py

```diff
@@ -532,7 +532,7 @@ def apply_async(self, args=None, kwargs=None, task_id=None, producer=None,
         return app.send_task(
             self.name, args, kwargs, task_id=task_id, producer=producer,
             link=link, link_error=link_error, result_cls=self.AsyncResult,
-            shadow=shadow, task_type=self,
+            shadow=shadow, task_type=self, ignore_result=getattr(self, 'ignore_result', False),
```

Comment:
I think this is a `Task` class attribute so you do not need `getattr`. Also, shouldn't this change be applied to `apply`?